tests/testPsfDetermination.py

```diff
@@ -219,7 +219,7 @@ def tearDown(self):
         del self.catalog
 
     @staticmethod
-    def setupDeterminer(exposure, nEigenComponents=3, starSelectorAlg="secondMoment"):
+    def setupDeterminer(exposure, schema, nEigenComponents=3, starSelectorAlg="secondMoment"):
```

Comment:
Please consider making this a normal method so that the schema argument can be eliminated or can default to self.catalog.schema (presumably by defaulting to None and if None then use self.catalog.schema).